setup.py

```diff
@@ -3,8 +3,7 @@
 
 ENTRY_POINTS = {
     'pipeline.handlers': [
-        'AatamsDmHandler = aodndata.aatams.aatams_dm:AatamsDmHandler',
-        'AatamsNrtHandler = aodndata.aatams.aatams_nrt:AatamsNrtHandler',
```

Comment:
since you removed them maybe you could delete the python files as well? see with @evacougnon

Reply:
as well as their respective tests.
something funny is happening with travis, could be related
https://travis-ci.org/github/aodn/python-aodndata/jobs/671226621

Reply:
This same PR needs to be made in aodndata: https://github.com/aodn/python-aodntools/pull/119

Some packages now need to be constrained on Python 3.5 environments (in this case cftime).

Reply:
>since you removed them maybe you could delete the python files as well? see with @evacougnon

Yeap - and this will probably break something in the other packages...

@evacougnon - Am I free to remove all remaining aatams stuff here?
 
yeah - I noticed yesterday that some free-range packages were breaking with 3.5

Reply:
> > since you removed them maybe you could delete the python files as well? see with @evacougnon
> 
> Yeap - and this will probably break something in the other packages...

what do you mean?

Reply:
my bad - I didn't want it to sound ironic! This means I agree, but we need to keep a track if there is something else still using/running, mostly because I can't remember the current AATAMS state of affairs.

Reply:
removal done